administration/src/cards/CardBlueprint.ts

```diff
@@ -102,7 +102,9 @@ export class CardBlueprint implements JSONCardBlueprint {
     return new CardInfo({
       fullName: this.fullName,
       expirationDay:
-        expirationDate !== null && !this.hasInfiniteLifetime() ? dateToDaysSinceEpoch(expirationDate) : undefined,
+        expirationDate !== null && Date.parse(expirationDate.toISOString()) > 0 && !this.hasInfiniteLifetime()
```

Comment:
why not 
```suggestion
        expirationDate !== null && expirationDate > 0 && !this.hasInfiniteLifetime()
```

Reply:
because i'm checking the milliseconds and not the date greater 0